apps/schematic/api/schematic_api/controllers/schema_controller_impl.py

```diff
@@ -141,73 +170,66 @@ def get_connected_node_pair_page(
 def get_node_is_required_from_schematic(
     node_display: str,
     schema_url: str,
+    display_label_type: DisplayLabelType = "class_label",
 ) -> bool:
     """Gets whether or not the node is required by the schema
 
     Args:
         node_display(str): The display name of the node
         schema_url (str): The URL of the schema in jsonld form
+        display_label_type (DisplayLabelType):
+           The type of label to use as display
+           Defaults to "class_label"
 
     Returns:
        bool: Whether or no the node is required
     """
-    schema_generator = SchemaGenerator(path_to_json_ld=schema_url)
-    return schema_generator.is_node_required(node_display)
+    data_model_parser = DataModelParser(path_to_data_model=schema_url)
+    parsed_data_model = data_model_parser.parse_model()
+    data_model_grapher = DataModelGraph(
+        attribute_relationships_dict=parsed_data_model,
+        data_model_labels=display_label_type,
+    )
+    graph_data_model = data_model_grapher.generate_data_model_graph()
```

Comment:
For this and anything else like it, the you can do `graph_data_model = data_model_grapher.graph` as its created in the `init` of `DataModelGraph`